downloads/swan-lake-release-notes/swan-lake-2201.4.0/RELEASE_NOTE.md

```diff
@@ -57,6 +57,62 @@ Annotations are now supported on tuple members. Tuple members can be annotated u
 
 To view other bug fixes, see the [GitHub milestone for Swan Lake 2201.4.0](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+label%3ATeam%2FCompilerFE+milestone%3A2201.4.0+is%3Aclosed+label%3AType%2FBug).
 
+#### Backward incompatible changes
+
+- Fixed a bug that incorrectly resolved the result type of a query action that would complete normally to `error?` instead of `()`.
+  Now, resolution of the result type of a query action has been improved so that it is now set to `error` only in instances where an error can be generated during the execution of the query pipeline (`from-clause`/`join-clause`).
```

Comment:
```suggestion
- Fixed a bug that incorrectly resolved the result type of a query action that would complete normally to `error?` instead of `()`. Now, the result type includes `error` only in instances where an error can be generated during the execution of the query pipeline (`from` clause/`join` clause).
```

Reply:
I don't think we need the separate lines (with the first line being like the title.).